plugin/trino-bigquery/src/test/java/io/trino/plugin/bigquery/TestBigQueryIntegrationSmokeTest.java

```diff
@@ -158,4 +161,37 @@ public void testShowCreateTable()
                         "   comment varchar NOT NULL\n" +
                         ")");
     }
+
+    @Test(dataProvider = "testTableNameDataProvider")
+    public void testCanSelectFromMixedCaseTables(String tableName)
```

Comment:
We typically test this with a dedicated test class, like TestPostgreSqlCaseInsensitiveMapping.

This also provdes for inspiration re: test cases  & structure.